spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/error/DefaultErrorAttributes.java

```diff
@@ -64,6 +66,12 @@
 	private static final String ERROR_ATTRIBUTE = DefaultErrorAttributes.class.getName()
 			+ ".ERROR";
 
+	private boolean includeException;
```

Comment:
Yes, I'd also prefer that. Just went with more passive approach initially, will update this.